Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:5
```

Comment:
We just need to test that it works on 6 and then bump.

I don't mind bumping versions, but it would be good to get this one merged soon.